lib/modules/manager/mint/extract.ts

```diff
@@ -0,0 +1,20 @@
+import { GitTagsDatasource } from '../../datasource/git-tags';
+import type { PackageDependency, PackageFile } from '../types';
+
+export function extractPackageFile(content: string): PackageFile | null {
+  const deps: PackageDependency[] = [];
+
+  content.split('\n').forEach((line) => {
+    if (line.includes('@')) {
```

Comment:
You need to handle comments `#`, they can be at line start or after a package.

 https://github.com/yonaskolb/Mint/blob/2f34a6d72c7d3543ef4a7f8f6f950c72eb32cce6/Sources/MintKit/Mintfile.swift#L20-L32

---

extract deps without version and add `skipReason`

https://github.com/renovatebot/renovate/blob/e79402d5a4cf5fdd8af0fbb98b71b80515aae107/lib/modules/manager/ansible-galaxy/collections.ts#L122-L124